repl/src/main/scala/com/cloudera/livy/repl/Session.scala

```diff
@@ -168,4 +195,21 @@ class Session(interpreter: Interpreter, stateChangedCallback: SessionState => Un
 
     compact(render(resultInJson))
   }
+
+  private def setJobGroup(statementId: Int): String = {
+    val cmd = Kind(interpreter.kind) match {
+      case Spark() | PySpark() | PySpark3() =>
+        s"""sc.setJobGroup("$statementId", "Job group for statement $statementId")"""
```

Comment:
This's weird. I tested this with Livy w/ Spark 2.0:
<img width="400" alt="screen shot 2016-12-14 at 1 04 38 am" src="https://cloud.githubusercontent.com/assets/8400790/21175920/76302e06-c199-11e6-9cd7-bef594c857f3.png">


```
     {
      "id": 5,
      "state": "available",
      "output": {
        "status": "ok",
        "execution_count": 5,
        "data": {
          "text/plain": "u'2.0.0.2.5.2.1-1'"
        }
      }
    },
    {
      "id": 6,
      "state": "available",
      "output": {
        "status": "ok",
        "execution_count": 6,
        "data": {
          "text/plain": "1"
        }
      }
    },
    {
      "id": 7,
      "state": "available",
      "output": {
        "status": "error",
        "execution_count": 7,
        "ename": "NameError",
        "evalue": "name 'res0' is not defined",
        "traceback": [
          "Traceback (most recent call last):\n",
          "NameError: name 'res0' is not defined\n"
        ]
      }
    }
```